vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/AbstractNumberField.java

```diff
@@ -94,12 +94,10 @@ public AbstractNumberField(SerializableFunction<String, T> parser,
         setValueChangeMode(ValueChangeMode.ON_CHANGE);
 
         addValueChangeListener(e -> validate());
-        addInvalidChangeListener(e -> {
-            // If invalid is updated from client to false, check it
-            if (e.isFromClient() && !e.isInvalid()) {
-                validate();
-            }
-        });
+
+        addAttachListener(e -> getElement()
+                .executeJs("$0.validate = function () {return this.checkValidity();}",
+                        getElement()));
```

Comment:
Can we now remove everything related to the `RequiredValidationUtil` (the helper class to disable client validation in some cases, inside text-field-flow repo)?